indexer/src/main/java/org/uniprot/store/indexer/crossref/readers/CrossRefReader.java

```diff
@@ -162,22 +162,16 @@ private CrossRefEntry convertToDBXRef(String linesStr) {
         builder.dbUrl(url).category(cat);
 
         // update the reviewed and unreviewed protein count
-        CrossRefUniProtCountReader.CrossRefProteinCount reviewedProtCount =
-                this.crossRefProteinCountMap.get(abbr + UNDER_ZERO);
```

Comment:
Can you please remove constant variable (UNDER_ZERO and UNDER_ONE) that are not being used...